gems/pending/Gemfile

```diff
@@ -54,18 +54,7 @@ if RbConfig::CONFIG["host_os"].include?("linux")
   end
 end
 
-# qpid group is needed to gate the inclusion of qpid_messaging gem on platforms
-# where the qpid-cpp-client-devel package is not available.  This includes
-# OSX, Windows, and CruiseControl machines.
-group :qpid do
-  gem "qpid_messaging", "~>0.20.0", :require => false
-end
-
-# Grouping this mainly to stay consistent with qpid--the other amqp library.  In
-# a production build, the qpid and rabbit groups should be included
-group :rabbit do
-  gem "bunny", "~>1.0.4", :require => false
-end
+gem "bunny", "~>1.0.4", :require => false
```

Comment:
[Bunny](https://github.com/ruby-amqp/bunny) is now well into `2.x` land.  We'll have to see if it's prudent to upgrade to a newer version.  But, this seems pretty old to me.